themes/theme-default/src/client/components/NavbarBrand.vue

```diff
@@ -48,6 +56,7 @@ const NavbarBrandLogo: FunctionalComponent = () => {
       v-if="navbarBrandTitle"
       class="site-name"
       :class="{ 'can-hide': navbarBrandLogo }"
+      :aria-hidden="navBarLogoAltMatchesTitle"
```

Comment:
I dislike the change. An empty alt for logo image would hide the alerts, and that's what my theme is doing (yes I made a lot of improvements downstream at theme hope but not syncing them back the default theme).


For this pr let's discuss 2 things:

1. In what situation, the site builder needs to provide a name for logo ranther than the site title he manually defined? (Logo alt is needed to be set manually instead of an empty alt (`alt=""`)

2. What's the benefit for letting the reader to read the logo and omit the actual title text, instead of reading the title text and omitting the logo?

Reply:
The current issue is the text is duplicated and thus read twice by screen readers.  I don't think the issue is about hiding the alert as much as the alerts actually point to bad behavior to some users (especially from Deque who has done more testing and research then any single person could around accessibility).

I decided to add the option for specifying different alt text so the site designer can decide what is best for them. I agree that most of the time the image alt text and the site title are going to be the same but not necessarily always. For example, I could see a situation where a company has a sub-title or slogan that they want read by a screen reader because its part of their logo but wouldn't want it displayed as the site title.

The benefit for reading the logo alt text instead of the title when they are the same is just that that was the easier and cleaner implementation. There is no difference to how its read by screen readers.